src/fiat-connect-client.ts

```diff
@@ -82,9 +83,18 @@ export class FiatConnectClient implements FiatConnectApiClient {
    * @returns a Promise resolving to the literal string 'success' on a
    * successful login or an Error response.
    */
-  async login(): Promise<Result<'success', ResponseError>> {
+  async login(params: LoginParams): Promise<Result<'success', ResponseError>> {
```

Comment:
```suggestion
  async login(params?: LoginParams): Promise<Result<'success', ResponseError>> {
```
nit. since it seems like the params are optional